control/tests/freqresp_test.py

```diff
@@ -508,3 +509,17 @@ def test_dcgain_consistency():
 
     sys_ss = ctrl.tf2ss(sys_tf)
     np.testing.assert_almost_equal(sys_ss.dcgain(), -1)
+
+
+def test_singular_values_plot():
```

Comment:
Please parametrize with additional:
* StateSpace systems 
* FRD objects.
* SISO 
* ctime and dtime
* non-square input, output MIMO

Reply:
Should I define these different systems for this test only?
Perhaps it could be convenient to define some of them as @pytest.fixture?

Reply:
Yes, please. See my comment above.